plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -783,12 +826,35 @@ public void testSystemMetadataTableComments()
                 assertUpdate(session, "INSERT INTO test_select_s_m_t_comments" + i + " VALUES ('xyz', 12)", 1);
 
                 assertUpdate(session, "CREATE TABLE test_other_select_s_m_t_comments" + i + "(id varchar, age integer)"); // won't match the filter
+
+                if (removeCachedProperties) {
```

Comment:
It took me a bit to understand the purpose of `removeCachedProperties` in `testSystemMetadataTableComments()` method.

I understand though now the purpose - we want to verify how the `system.metadata.table_comments` query behaves when the table comments are added filled one by one as well as when they are added in bulk.